src/containers/Accounts/AMM/AMMAccounts/index.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { useParams, useRouteMatch } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import { connect } from 'react-redux'
+import AMMAccountHeader from 'containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader'
+import { AccountTransactionTable } from 'containers/Accounts/AccountTransactionTable/index'
+import NoMatch from 'containers/NoMatch'
+import 'containers/Accounts/styles.scss'
+import {
+  analytics,
+  ANALYTIC_TYPES,
+  formatAsset,
+  formatBalance,
+} from 'containers/shared/utils'
+import { Tabs } from 'containers/shared/components/Tabs'
+import { AccountAssetTab } from 'containers/Accounts/AccountAssetTab/AccountAssetTab'
+import { getAccountTransactions, getAMMInfo } from 'rippled/lib/rippled'
+import SocketContext from '../../../shared/SocketContext'
+import { ERROR_MESSAGES } from '../../Errors'
+import { getTokenPairData } from '../../../../apis/OnTheDex'
```

Comment:
This file does not exist.

Reply:
It exists in the other PR: https://github.com/ripple/explorer/pull/498

Reply:
removed it